File: src/main/java/com/github/redfoos/logstash/psi/LogstashElementType.java
package com.github.redfoos.logstash.psi;

import com.github.redfoos.logstash.LogstashLanguage;
import com.intellij.psi.tree.IElementType;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class LogstashElementType extends IElementType {
    public LogstashElementType(@NotNull @NonNls String debugName) {
        super(debugName, LogstashLanguage.INSTANCE);
    }
}

File: src/main/kotlin/com/github/redfoos/logstash/psi/LogstashElementType.kt
package com.github.redfoos.logstash.psi

import com.github.redfoos.logstash.LogstashLanguage
import com.intellij.psi.tree.IElementType

class LogstashElementType(debugName: String): IElementType(debugName, LogstashLanguage.INSTANCE)
